### PR TITLE
Add Android support

### PR DESCRIPTION
**Pull request by KGronek-Pubnub**
build(cmake): Add Android support

Prepare CMakeLists to support builds for Arm64 architecture

fix(cmake): Fix build for Windows

Fix CMakeLists to build correctly on Windows

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -445,8 +445,13 @@ if(${OPENSSL})
             if(WIN32 OR WIN64 OR MSVC)
                 find_library(OpenSSL REQUIRED NAMES libssl PATHS ${OPENSSL_ROOT_DIR}/${CUSTOM_OPENSSL_LIB_DIR} NO_DEFAULT_PATH)
                 find_library(Crypto REQUIRED NAMES libcrypto PATHS ${OPENSSL_ROOT_DIR}/${CUSTOM_OPENSSL_LIB_DIR} NO_DEFAULT_PATH)
-            else()
-                find_library(OpenSSL REQUIRED NAMES ssl PATHS ${OPENSSL_ROOT_DIR}/${CUSTOM_OPENSSL_LIB_DIR} NO_DEFAULT_PATH)
+            elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|arm64")
+				# find_library doesn't work when building for Android architecture
+				set(OpenSSL "${OPENSSL_ROOT_DIR}/${CUSTOM_OPENSSL_LIB_DIR}/libssl.a")
+				set(Crypto "${OPENSSL_ROOT_DIR}/${CUSTOM_OPENSSL_LIB_DIR}/libcrypto.a")
```
Comment (Xavrax) [inline CMakeLists.txt:451]: strange formatting here. 
Do you use 4 spaces or tabs?

**Review by Xavrax — APPROVED** [review]
LGTM - just fix formatting

**Comment by KGronek-Pubnub** [comment]
@pubnub-release-bot release

**Comment by KGronek-Pubnub** [comment]
@parfeon Thanks for fixing the tests. It looks like release command doesn't work as well?

**Comment by KGronek-Pubnub** [comment]
@pubnub-release-bot release

**Comment by KGronek-Pubnub** [comment]
@parfeon Now the command worked, but it looks like it overrode the last release instead of creating new one.

**Comment by parfeon** [comment]
@KGronek-Pubnub because PR message wasn't properly formatted :(

**Comment by KGronek-Pubnub** [comment]
@pubnub-release-bot release

**Pull request merged**